test/sample.cc

```diff
@@ -186,6 +191,38 @@ TEST(Sample, hgt_file_name) {
   ASSERT_EQ(testable_sample_t::get_hgt_file_name(amsterdam_index), "/N52/N52E004.hgt");
 }
 
+TEST(Sample, test_make_single_point_url) {
+  struct test_case {
+    std::string url;
+    midgard::PointLL point;
+    std::string result;
+  };
+
+  std::vector<test_case> tests{{{"http://s3.amazonaws.com/mapzen.valhalla{DataPath}"},
+                                {-73.512143, 40.646556},
+                                {"http://s3.amazonaws.com/mapzen.valhalla/N40/N40W074.hgt"}},
+                               {{"http://s3.amazonaws.com/mapzen.valhalla{DataPath}"},
+                                {-78.504476, -0.212297},
+                                {"http://s3.amazonaws.com/mapzen.valhalla/S01/S01W079.hgt"}},
+                               {{"http://s3.amazonaws.com/mapzen.valhalla{DataPath}"},
+                                {152.967888, -27.533658},
+                                {"http://s3.amazonaws.com/mapzen.valhalla/S28/S28E152.hgt"}},
+                               {{"http://s3.amazonaws.com/mapzen.valhalla{DataPath}"},
+                                {139.737345, 35.628096},
+                                {"http://s3.amazonaws.com/mapzen.valhalla/N35/N35E139.hgt"}},
+                               {{"http://s3.amazonaws.com/mapzen.valhalla{DataPath}"},
+                                {4.898484, 52.380697},
+                                {"http://s3.amazonaws.com/mapzen.valhalla/N52/N52E004.hgt"}}};
+
```

Comment:
i mean this is great for test coverage but... its pretty expensive for a unit test right? off the top of my head i think each of these is like 25 megabytes in uncompressed form.

Reply:
well... i just consider more tests better than more space. space is cheap.

Reply:
@kevinkreiser, these are not requests to a remote storage. this test is purely for checking if the function which substitutes "{DataPath}" with elevation tiles path works correctly.

Reply:
ah ok sorry, i was trying to do a quick pass on the review!